data/summative-questionnaires/personal-social-and-emotional-development.yml

```diff
@@ -222,16 +222,13 @@ personal-social-and-emotional-development:
       1_visual-reminders:
         multi_select: false
         label: |
-          True or false? Visual reminders will positively support children to develop
+          True or false? Quiet activities will positively support children to develop
```

Comment:
should this true or false be on a separate line to the statement that comes after? just noticed on the other commit they were

Reply:
They should. Put on separate lines.